visivo/models/model.py

```diff
@@ -141,6 +141,40 @@ def insert_csv_to_sqlite(self, output_dir):
         data_frame.to_sql(self.table_name, engine, if_exists="replace", index=False)
 
 
+class LocalMergeModel(Model, ParentModel):
+    """
+    Local Merge Models are models that allow you to merge data from multiple other models locally.
```

Comment:
@tgsoverly Let me know if you want me to write a doc string for this.

Reply:
Sure if you want to update it!